src/js/core/methods/LiskGetAddress.js

```diff
@@ -0,0 +1,127 @@
+/* @flow */
+'use strict';
+
+import AbstractMethod from './AbstractMethod';
+import { validateParams } from './helpers/paramsValidator';
+import { validatePath, fromHardened, getSerializedPath } from '../../utils/pathUtils';
+
+import * as UI from '../../constants/ui';
+import { UiMessage } from '../../message/builder';
+
+import type { UiPromiseResponse } from 'flowtype';
+import type { LiskAddress } from '../../types/trezor';
+import type { CoreMessage } from '../../types';
+import type { LiskAddress as LiskAddressResponse } from '../../types/lisk';
+
+type Batch = {
+    path: Array<number>,
+    showOnTrezor: boolean,
+}
+
+type Params = {
+    bundle: Array<Batch>,
+    bundledResponse: boolean,
+}
+
+export default class LiskGetAddress extends AbstractMethod {
+    params: Params;
+    confirmed: boolean = false;
+
+    constructor(message: CoreMessage) {
+        super(message);
+
+        this.requiredPermissions = ['read'];
+        this.requiredFirmware = ['0', '2.0.8'];
```

Comment:
Lisk is already supported by Trezor T from`2.0.7` firmware version. Why the first parameter is `0`?

Reply:
first parameter is a firmware for Trezor 1 second is for TrezorT. There is a '0' since there is no support for Trezor1, i will change support for TrezorT to '2.0.7', thanks!

Reply:
support for Trezor 1 is planed in `1.0.7` version based on this milestone https://github.com/trezor/trezor-mcu/milestone/2. PR for Trezor 1 is waiting for review https://github.com/trezor/trezor-mcu/pull/351

Reply:
great, i will add this as well, thanks